wdl/src/main/scala/wdl/expression/FileEvaluator.scala

```diff
@@ -34,9 +34,8 @@ class FileEvaluatorWdlFunctions
  * evaluate("b.txt") would return Seq().
  */
 case class FileEvaluator(valueEvaluator: ValueEvaluator, coerceTo: WomType = WomAnyType) {
-  type T = Seq[WomFile]
-  def lookup = (s:String) => Seq.empty[WomFile]
 
+  def lookup = (s:String) => Seq.empty[WomFile]
```

Comment:
Would this work like this? Feels nicer to me...:
```scala
def lookup: String => Seq[WomFile] = _ => Seq.empty
```

Reply:
Hm.  Not sure why this is highlighted, I didn't change this line.  I deleted the one above it.

(FWIW I agree)